src/Set/FOSRestSetList.php

```diff
@@ -1,18 +0,0 @@
-<?php
```

Comment:
@TomasVotruba this removal seems cause eror on rector-phpunit rector run:

https://github.com/rectorphp/rector-phpunit/actions/runs/17376070939/job/49322564263?pr=527#step:5:13

```
 [ERROR] Class "Rector\Symfony\Set\FOSRestSetList" not found                    
```

Reply:
Removing usage at rector-src

- https://github.com/rectorphp/rector-src/pull/7204